mamoru/tracer.go

```diff
@@ -0,0 +1,84 @@
+package mamoru
+
+import (
+	"math/big"
+	"os"
+	"time"
+
+	"github.com/Mamoru-Foundation/mamoru-sniffer-go/evm_types"
+	"github.com/Mamoru-Foundation/mamoru-sniffer-go/mamoru_sniffer"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+var sniffer *mamoru_sniffer.Sniffer
+
+func init() {
+	if IsSnifferEnable() {
+		defer func() {
+			if r := recover(); r != nil {
+				log.Error("Mamoru Sniffer Recovered:", r)
+			}
+		}()
+		var err error
+		sniffer, err = mamoru_sniffer.Connect()
+		if err != nil {
+			log.Error("Mamoru Sniffer connect error:", err)
+			panic(err)
```

Comment:
What the reason to panic here and recover right after? What happens if we just remove `panic(err)` ?

Reply:
You are right, panic is not needed, the code does not even reach it and the recovery does not work out